x-pack/legacy/plugins/siem/public/components/charts/areachart.test.tsx

```diff
@@ -201,26 +201,8 @@ describe('AreaChartWithCustomPrompt', () => {
           color: '#490092',
         },
       ],
-      [
```

Comment:
@elastic/siem This chart is passed as the second argument to the function below, meaning it's not used, and it doesn't pass as its own test, so I removed it.

Reply:
The purpose of this case should be the same as the one below, I would wish to keep it here if it is ok.

Reply:
We're going to fix this in a follow up PR along with https://github.com/elastic/kibana/pull/46391#discussion_r327787164

Reply:
We're going to fix this in a follow up pr lik https://github.com/elastic/kibana/pull/46391#discussion_r327787164